Connected agent - clarification on disabling the transaction report (#2009)
Axway/axway-open-docs: content/en/docs/central/connected_agent_common_reference/traceability_usage.md

The Traceability Agent is attached to a Gateway and monitors the traffic crossing it. The collected traffic is reported to Amplify platform in different events:

> * **Usage** event: report the total number of API calls during a period of time. This feature cannot be inactivated.

Reviewer: ```suggestion
* **Usage** event: report the total number of API calls during a period of time. This feature cannot be inactivated.
```
```suggestion
* **Usage** event: reports the total number of API calls during a period of time. This feature cannot be inactivated.
```

> * **Transaction** event: report the transaction summary (API name, duration, status), as well as the transaction details (request/response headers from the frontend and backend of the API). Traceability agent is also able to report JMS property if your API call some JMS endpoint. You can reduce the number of transactions sent to the platform using the [sampling feature](/docs/central/connected_agent_common_reference/trace_sampling).

Reviewer: ```suggestion
* **Transaction** event: report the transaction summary (API name, duration, status), as well as the transaction details (request/response headers from the frontend and backend of the API). Traceability agent is also able to report JMS property if your API call some JMS endpoint. You can reduce the number of transactions sent to the platform using the [sampling feature](/docs/central/connected_agent_common_reference/trace_sampling).
```
```suggestion
* **Transaction** event: reports the transaction summary (API name, duration, status), and the transaction details (request/response headers from the frontend and backend of the API). If your API calls JMS endpoints, the JMS properties are also reported. To reduce the number of transactions sent to the platform, use the [sampling feature](/docs/central/connected_agent_common_reference/trace_sampling).
```

* **Usage** event: report the total number of API calls during a period of time. This feature cannot be inactivated.
* **Transaction** event: report the transaction summary (API name, duration, status), as well as the transaction details (request/response headers from the frontend and backend of the API). Traceability agent is also able to report JMS property if your API call some JMS endpoint. You can reduce the number of transactions sent to the platform using the [sampling feature](/docs/central/connected_agent_common_reference/trace_sampling).

> {{< alert title="Disabling transacation report" color="warning" >}}If you want to use the Traceability agent for logging usage only, then you do not need to report transactions and you should set `TRACEABILITY_SAMPLING_PERCENTAGE=0` and `TRACEABILITY_SAMPLING_REPORTALLERRORS=false`. These 2 properties will tell the Traceability agent to disregard the transaction report. Once the property are set in `ta_env_vars.env` file, the traceability agent must be restarted to use the new configuration.{{< /alert >}}

Reviewer: ```suggestion
{{< alert title="Disabling transacation report" color="warning" >}}To use the Traceability Agent for logging usage only, set `TRACEABILITY_SAMPLING_PERCENTAGE=0` and `TRACEABILITY_SAMPLING_REPORTALLERRORS=false` in the `ta_env_vars.env` file to disable the transaction report. Restart the Traceability Agent to use the new configuration.{{< /alert >}}
```
